src/main/java/org/opensearch/knn/plugin/rest/RestDeleteModelHandler.java

```diff
@@ -0,0 +1,60 @@
+package org.opensearch.knn.plugin.rest;
+
+import com.google.common.collect.ImmutableList;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.common.Strings;
+import org.opensearch.knn.plugin.KNNPlugin;
+import org.opensearch.knn.plugin.transport.DeleteModelAction;
+import org.opensearch.knn.plugin.transport.DeleteModelRequest;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+import org.opensearch.rest.action.admin.cluster.RestNodesUsageAction;
+
+import java.util.List;
+import java.util.Locale;
+
+import static org.opensearch.knn.common.KNNConstants.MODELS;
+import static org.opensearch.knn.common.KNNConstants.MODEL_ID;
+
+public class RestDeleteModelHandler extends BaseRestHandler {
+
+    public static final String NAME = "delete_knn_model_action";
```

Comment:
prefix delete with knn as was done for get

Reply:
Ack. Updated it.